municipal-services/sw-calculator/src/main/java/org/egov/swcalculation/service/SWCalculationServiceImpl.java

```diff
@@ -102,7 +102,7 @@ public List<Calculation> getCalculation(CalculationReq request) {
 					request.getIsconnectionCalculation());
 			unsetSewerageConnection(calculations);
 		}
-		else if (request.getIsReconnectionRequest())	
+		if (request.getIsReconnectionRequest())
```

Comment:
Why else condition is removed?

Reply:
it is also repeated code.I removed it once check it.